pkg/cincinnati/cincinnati.go

```diff
@@ -249,6 +262,47 @@ func (c Client) CalculateUpgrades(ctx context.Context, uri *url.URL, arch, sourc
 	return current, requested, upgrades, err
 }
 
+// getSemverFromChannel will return the major and minor version from the source and target channels. The prefix returned is
+// for the source channels for cross channel calculations.
+func getSemverFromChannels(sourceChannel, targetChannel string) (source, target semver.Version, prefix string, err error) {
```

Comment:
This is one ugly semver dissection operation. I understand this is how it is with Cincinnati but something should be done somewhere to make this cleaner on either in Cincinnati itself or on our side. Perhaps, this is something for another PR.